build.rs

```diff
@@ -52,68 +53,58 @@ fn build() {
     platform::print_libs(out_dir);
 }
 
+// Similar to unwrap, but panics on just the error value
+#[allow(dead_code)]
+fn err_to_panic<T, E: Display>(result: Result<T, E>) -> T {
+    match result {
+        Ok(x) => x,
+        Err(e) => panic!("{}", e)
+    }
+}
```

Comment:
I think this is just the same as calling `.unwrap()` on the Result right? Can we do that instead? I'm not sure why the existing ones were matching and `panic!`ing inline.